programmers/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,22 @@
+function solution(s){
+    s = s.toLowerCase();
+    let p = s.match(/p/g);
+    let y = s.match(/y/g);
+    
+    if(p !=null && y != null){
+      if(p.length == y.length){
+         return true;
+      }
+      else if(p.length != y.length){
+        return false;
+        
+      }
+    }
```

Comment:
1. if(공백)(조건)(공백){ ... } 식으로 작성해주세요. for문도 마찬가지 입니다. for(공백)(let i = 0; i < n; i++)(공백){ ... }

2. 자바스크립트에서는 ==연산자와 ===연산자가 있습니다. ==연산자는 동등 연산자이고 ===연산자는 일치연산자 인데 ===가 좀 더 정확하게 비교를 해주기때문에 특별한 상황 아니면 ===연산자를 이용해주세요.

    ==, ===연산자 차이: http://guswnsxodlf.github.io/javascript-equal-operator

```suggestion
    if (p !== null && y !== null) {
      return p.length === y.length
    }
```